### ISSUE TITLE
Extend NFS Server to add exports

### ISSUE THREAD

**Issue by td512** [ShellManager/roles#9]
The NFS server playbook found in the nfs-server branch needs to be updated to support automated provisioning of exports

**Comment by td512** [comment]
This one should be relatively simple @Keanu73. I was thinking along the lines of `path` for the path of the export, `allowed` for the allowed subnet, `permissions` for the permissions (like `ro` or `rw`), and `create` to specify whether or not the directory should be created

**Comment by td512** [comment]
Sample role.yml

```
---
paths:
  - { path: /var, permissions: ro, subnet: "*", create: no }
  - { path: /home, permissions: rw,sync,no_root_squash,no_all_squash, subnet: "10.10.10.1/32", create: no }
  - { path: /xyz, permissions: ro, subnet: "1.1.1.1/32", create: yes }
```

**Comment by td512** [comment]
Solved by #8
